### PR TITLE
Display severity in human format results

### PR DESCRIPTION
**Pull request by nvuillam**
- Severity in flow errors details
- Summary of number of issues by severity
- yarn.lock has been updated after I run "yarn" at the root, and i'm using yarn latest version so I think it's correct

Screenshot of the result :)

image

### CODE REVIEW

**Comment by nvuillam** [comment]
@RubenHalman FYI :)

**Comment by RubenHalman** [comment]
@nvuillam Awesome! If you can please leave out the changes to the yarn lock file, we can merge this!

File: yarn.lock
Comment (RubenHalman) [inline yarn.lock:1]: Please leave the yarn changes out of the PR

**Review by RubenHalman — CHANGES_REQUESTED** [review]
@nvuillam Awesome! If you can please leave out the changes to the yarn lock file, we can merge this immidiately!

**Comment by nvuillam** [comment]
@RubenHalman done :)

**Review by RubenHalman — APPROVED** [review]

**Comment by RubenHalman** [comment]
Thank you for your contribution. This change is now included in version [2.19](https://github.com/Lightning-Flow-Scanner/lightning-flow-scanner-sfdx/releases/tag/2.19)

**Pull request merged**